src/app/phase2/issues-faulty/issues-faulty.component.ts

```diff
@@ -24,11 +25,9 @@ export class IssuesFaultyComponent implements OnInit, OnChanges {
   @ViewChild(MatPaginator) paginator: MatPaginator;
 
   constructor(public issueService: IssueService, private errorHandlingService: ErrorHandlingService, public userService: UserService,
-    public permissions: PermissionService, private labelService: LabelService) {
+    public permissions: PermissionService, private labelService: LabelService, private githubService: GithubService) {
     if (userService.currentUser.role === UserRole.Student) {
       this.displayedColumns = ['id', 'title', 'type', 'severity', 'responseTag', 'assignees', 'duplicatedIssues', 'actions'];
-    } else if (userService.currentUser.role === UserRole.Tutor) {
-      this.displayedColumns = ['id', 'title', 'teamAssigned', 'type', 'severity', 'responseTag', 'assignees', 'duplicatedIssues'];
```

Comment:
Will tutors have write access to Team's repo during the Team's response phase?

Reply:
Um, team's repo should not be involved at all. The bug reports will be in a central repo during the dev-response phase, as before. Tutors need not have write access to it.

Reply:
Ohh right, I have mistaken it for the long term goal. But either way, Tutors will not have write access during the dev-response phase. So I don't think providing Tutors the UI to edit the issues is appropriate.

Reply:
Ohh wait haha nvm, on a closer look the UI for editing issues are not rendered for tutors.